contracts/interchain-token-service/src/contract.rs

```diff
@@ -50,49 +53,50 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
         env.storage().instance().get(&DataKey::GasService).unwrap()
     }
 
-    fn its_hub_routing_identifier(env: &Env) -> String {
-        String::from_str(env, ITS_HUB_ROUTING_IDENTIFIER)
+    fn its_hub_address(env: &Env) -> String {
+        env.storage().instance().get(&DataKey::ItsHub).unwrap()
     }
 
     fn its_hub_chain_name(env: &Env) -> String {
         String::from_str(env, ITS_HUB_CHAIN_NAME)
     }
 
-    fn trusted_address(env: &Env, chain: String) -> Option<String> {
+    fn trusted_chain(env: &Env, chain: String) -> bool {
```

Comment:
```suggestion
    fn is_trusted_chain(env: &Env, chain: String) -> bool {
```
it's now a validation vs a getter